packages/grafbase-nextjs-plugin/src/index.ts

```diff
@@ -13,21 +16,28 @@ const runGrafbase = () => {
 const createGrafbasePlugin =
   () =>
   (nextConfig: Partial<NextConfig> = {}): Partial<NextConfig> => {
-    // could be either `next dev` or just `next`
     const isNextDev =
       process.argv.includes('dev') ||
       process.argv.some(
         (_) => _.endsWith('bin/next') || _.endsWith('bin\\next')
       )
 
+    const hasLocalApiUrl =
+      (process.env['GRAFBASE_API_URL'] &&
+        (process.env['GRAFBASE_API_URL'].startsWith('http://localhost') ||
+          process.env['GRAFBASE_API_URL'].startsWith('http://127.0.0.1'))) ||
+      (process.env['NEXT_PUBLIC_GRAFBASE_API_URL'] &&
+        (process.env['NEXT_PUBLIC_GRAFBASE_API_URL'].startsWith(
+          'http://localhost'
+        ) ||
+          process.env['NEXT_PUBLIC_GRAFBASE_API_URL'].startsWith(
+            'http://127.0.0.1'
+          )))
```

Comment:
Could we use something like?
```ts
const GRAFBASE_API_URL = process.env['GRAFBASE_API_URL'] || process.env['NEXT_PUBLIC_GRAFBASE_API_URL']
const hasLocalApiUrl = /^http:\/\/(localhost|127\.0\.0\.1)/.test(GRAFBASE_API_URL)
```

Reply:
Yep. Great suggestion. I should have better refactored after dumping my previous POC.